resources/packs/os/smbios/linux.go

```diff
@@ -26,7 +26,7 @@ func (s *LinuxSmbiosManager) Info() (*SmBiosInfo, error) {
 	root := "/sys/class/dmi/id/"
 
 	wErr := afs.Walk(root, func(path string, info os.FileInfo, fErr error) error {
-		if fErr != nil {
+		if fErr != nil || os.IsPermission(fErr) {
```

Comment:
Should we somehow log the error, so users know why there is no data?
This is how it looks like for ports:
```
cnquery run -c "ports.listening" 
→ no provider specified, defaulting to local.
  Use --help for a list of available providers.
→ loaded configuration from /etc/opt/mondoo/mondoo.yml using source default
→ discover related assets for 1 asset(s)
→ resolved assets resolved-assets=1
! mql[processes]> could not retrieve process information error="process 782341 does not exist: /proc/782341" pid=782341
Query encountered errors:
1 error occurred:
	* cannot read related processess: permission denied
ports.listening: [
  0: port port=53 protocol="tcp4" address="127.0.0.53" process.executable=no data available
  1: port port=2222 protocol="tcp4" address="127.0.0.1" process.executable=no data available
  2: port port=53 protocol="tcp4" address="192.168.122.1" process.executable=no data available
  3: port port=631 protocol="tcp4" address="127.0.0.1" process.executable=no data available
  4: port port=6463 protocol="tcp4" address="127.0.0.1" process.executable="Discord"
  5: port port=111 protocol="tcp4" address="0.0.0.0" process.executable=no data available
  6: port port=631 protocol="tcp6" address="[::1]" process.executable=no data available
  7: port port=111 protocol="tcp6" address="[::]" process.executable=no data available
]
```

Reply:
nvm... we don't want to do this